src/main/kotlin/com/zepben/evolve/cim/iec61970/base/core/Curve.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61970.base.core
+
+import com.zepben.evolve.services.common.extensions.asUnmodifiable
+
+
+/**
+ * The Curve class is a multipurpose functional relationship between an independent variable (X-axis) and dependent (Y-axis) variables.
+ *
+ * @property _curveDatas The point data values that define this curve.
```

Comment:
unnecessary to document hidden fields - the documentation will be on the public `data` field.

Reply:
removed